staging/publishing/rules.yaml

```diff
@@ -2392,4 +2392,4 @@ rules:
       - staging/src/k8s.io/endpointslice
 recursive-delete-patterns:
 - '*/.gitattributes'
-default-go-version: 1.21.6
+default-go-version: 1.22rc2
```

Comment:
does this change the go version we publish to modules for patch tags? If we, we want to keep this at 1.21.x, not an rc